frontend/src/services.js/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+
+const baseUrl = "http://localhost:8000/api";
+
+export const getCourses = async() => {
+    try{
+        const response = await axios.get(`${baseUrl}/course/public`);
+        return response.data
+    }catch(error){
+        console.log(error)
```

Comment:
We still need to return something if it fails! Also, we can do `console.error` instead which will print it in red and flag the error in our console.

Reply:
I am now returning a custom error response that is handled async/await function in the front page.